spec/controllers/storage_controller_spec.rb

```diff
@@ -73,6 +73,9 @@
         login_as FactoryGirl.create(:user, :features => button)
 
         host = FactoryGirl.create(:host)
+        command = button.split('_', 2)[1]
+        Host.any_instance.stub(:is_available?).with(command).and_return(true)
```

Comment:
Why fake it with stubbing?  You could just use a real subclass of Host, I would think.